changelog.d/20230217_144946_joris_cursor_preserves_transaction_mode.md

```diff
@@ -0,0 +1,25 @@
+<!--
+A new scriv changelog fragment.
+
+Uncomment the section that is right (remove the HTML comment wrapper).
+-->
+
+
+### Patch
```

Comment:
Isn't this a breaking change?

Reply:
I wasn't sure myself, but I'll change it to be breaking